### PR TITLE
test: enable skipped tests

### PR DESCRIPTION
**Pull request by OrKoN**

### CODE REVIEW

**Comment by OrKoN** [comment]
cc @whimboo

File: test/TestExpectations.json [hunk 1]
```diff
@@ -4279,7 +4279,7 @@
     "testIdPattern": "[target.spec] Target should be able to use async waitForTarget",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["firefox", "webDriverBiDi"],
-    "expectations": ["SKIP"],
+    "expectations": ["PASS"],
```
Comment (OrKoN) [inline test/TestExpectations.json]:
```suggestion
    "expectations": ["SKIP"],
```

File: test/TestExpectations.json [hunk 2]
```diff
@@ -4352,7 +4352,7 @@
     "testIdPattern": "[target.spec] Target should not crash while redirecting if original request was missed",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["firefox", "webDriverBiDi"],
-    "expectations": ["SKIP"],
+    "expectations": ["PASS"],
```
Comment (OrKoN) [inline test/TestExpectations.json]:
```suggestion
    "expectations": ["SKIP"],
```

File: test/TestExpectations.json [hunk 3]
```diff
@@ -4383,7 +4383,7 @@
     "testIdPattern": "[target.spec] Target should report when a new page is created and closed",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["firefox", "webDriverBiDi"],
-    "expectations": ["SKIP"],
+    "expectations": ["PASS"],
```
Comment (OrKoN) [inline test/TestExpectations.json]:
```suggestion
    "expectations": ["SKIP"],
```

File: test/TestExpectations.json [hunk 4]
```diff
@@ -3204,19 +3204,19 @@
...
     "parameters": ["firefox", "webDriverBiDi"],
-    "expectations": ["SKIP"]
+    "expectations": ["PASS"]
   },
   {
     "testIdPattern": "[navigation.spec] navigation Page.waitForNavigation should work when subframe issues window.stop()",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["chrome", "webDriverBiDi"],
-    "expectations": ["SKIP"]
+    "expectations": ["PASS"]
```
Comment (OrKoN) [inline test/TestExpectations.json]:
```suggestion
    "expectations": ["SKIP"]
```

File: test/TestExpectations.json [hunk 5]
```diff
@@ -3234,13 +3234,13 @@
     "testIdPattern": "[navigation.spec] navigation Page.waitForNavigation should work with history.pushState()",
     "platforms": ["linux"],
     "parameters": ["firefox", "webDriverBiDi"],
-    "expectations": ["SKIP"]
+    "expectations": ["PASS"]
```
Comment (OrKoN) [inline test/TestExpectations.json]:
```suggestion
    "expectations": ["SKIP"]
```

File: test/TestExpectations.json [hunk 6]
```diff
@@ -3096,7 +3096,7 @@
     "testIdPattern": "[navigation.spec] navigation Page.goto should navigate to dataURL and fire dataURL requests",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["firefox", "webDriverBiDi"],
-    "expectations": ["SKIP"]
+    "expectations": ["PASS"]
```
Comment (OrKoN) [inline test/TestExpectations.json]:
```suggestion
    "expectations": ["SKIP"]
```

File: test/TestExpectations.json [hunk 7]
```diff
@@ -3204,19 +3204,19 @@
     "testIdPattern": "[navigation.spec] navigation Page.waitForNavigation should work when subframe issues window.stop()",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["firefox", "webDriverBiDi"],
-    "expectations": ["SKIP"]
+    "expectations": ["PASS"]
```
Comment (OrKoN) [inline test/TestExpectations.json]:
```suggestion
    "expectations": ["SKIP"]
```

File: test/TestExpectations.json [hunk 8]
```diff
@@ -2982,13 +2982,13 @@
     "testIdPattern": "[launcher.spec] Launcher specs Puppeteer Puppeteer.launch tmp profile should be cleaned up",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["firefox", "webDriverBiDi"],
-    "expectations": ["SKIP"]
+    "expectations": ["PASS"]
```
Comment (OrKoN) [inline test/TestExpectations.json]:
```suggestion
    "expectations": ["SKIP"]
```

File: test/TestExpectations.json [hunk 9]
```diff
@@ -4528,7 +4528,7 @@
     "testIdPattern": "[waittask.spec] waittask specs Frame.waitForSelector should survive cross-process navigation",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["firefox", "webDriverBiDi"],
-    "expectations": ["SKIP"]
+    "expectations": ["PASS"]
```
Comment (OrKoN) [inline test/TestExpectations.json]:
```suggestion
    "expectations": ["SKIP"]
```

File: test/TestExpectations.json [hunk 10]
```diff
@@ -3240,7 +3240,7 @@
     "testIdPattern": "[navigation.spec] navigation Page.waitForNavigation should work with history.replaceState()",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["firefox", "webDriverBiDi"],
-    "expectations": ["SKIP"],
+    "expectations": ["PASS"],
```
Comment (OrKoN) [inline test/TestExpectations.json]:
```suggestion
    "expectations": ["SKIP"],
```

**Review by whimboo — APPROVED** [review]
Nice!

**Comment by whimboo** [comment]
@OrKoN my patches on https://bugzilla.mozilla.org/show_bug.cgi?id=1877469 should help with flakiness as well.

**Comment by whimboo** [comment]
> @OrKoN my patches on https://bugzilla.mozilla.org/show_bug.cgi?id=1877469 should help with flakiness as well.

FYI due to issues on Android the landing of the patch will be delayed.

**Pull request merged**